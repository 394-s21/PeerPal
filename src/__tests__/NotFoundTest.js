import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import {BrowserRouter as Router, Route, Switch} from 'react-router-dom'; 
import NotFound from '../screens/NotFound';

test('Not Found Renders Correctly', () => {
  const page = renderer.create(

    <Router>
    <Route path="*">
        <NotFound></NotFound>
    </Route>
    </Router>
 ).toJSON();
  expect(page).toMatchSnapshot();
});
