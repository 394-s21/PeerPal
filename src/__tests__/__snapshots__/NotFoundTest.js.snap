// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Renders Correctly 1`] = `
<div
  className="notFound"
>
  <h2>
    Sorry
  </h2>
  <p>
    That page cannot be found
  </p>
  <a
    href="/"
    onClick={[Function]}
  >
    Back to the homepage...
  </a>
</div>
`;
